dev/Materials/Acrylic/AcrylicBrush_rs1_themeresources.xaml

```diff
@@ -48,7 +48,9 @@
             <StaticResource x:Key="AcrylicBackgroundFillColorDefaultBrush" ResourceKey="SystemChromeMediumLowColor" />
             <StaticResource x:Key="AcrylicInAppFillColorDefaultBrush" ResourceKey="SystemChromeMediumLowColor" />
             <SolidColorBrush x:Key="AcrylicBackgroundFillColorDefaultInverseBrush" Color="#F2F2F2" />
+            <SolidColorBrush x:Key="AcrylicInAppFillColorDefaultInverseBrush" Color="#F2F2F2" />
             <StaticResource x:Key="AcrylicBackgroundFillColorBaseBrush" ResourceKey="SystemChromeMediumLowColor" />
+            <StaticResource x:Key="AcrylicInAppFillColorBaseBrush" ResourceKey="SystemChromeMediumLowColor" />
```

Comment:
Shouldn't we add 4 brushes here too if we add 4 in RS2+? Though we hardly need to modify this file anyway since WinUI dropped RS1 support some time ago.

Reply:
@YuliKl what do you think we should do for RS1 regarding acrylic resources? Marcel is asking about the accent acrylics but RS1 doesn't even support acrylic.

Reply:
WinUI dropped supporting RS1, do we still need to update the corresponding files along with the supported versions?:)

Reply:
Historically we would have added acrylic brushes as solid color brushes downlevel, but agree with @beervoley that RS1 is no longer supported. Our [docs](https://github.com/microsoft/microsoft-ui-xaml#version-support) specify "TargetPlatformMinVersion >= 10.0.15063.0", which is RS2. @ranjeshj ?

Reply:
Should we then just get rid of the RS1 files to simplify our code base a bit and maybe reduce our compile time by a few seconds (separate PR of course)?

Reply:
I feel like we had this conversation already, but I can't recall what the result was. @jevansaks @kaiguo, I think we discussed this, do you recall what was decided upon?

Reply:
I don't recall the specifics of that conversation. Yes removing RS1 stuff from the repo sounds fine to me. We haven't officially supported or even tested on RS1 for quite some time.

Reply:
Just found the related issue for that (#1375). So just wait for WinUI 3 to drop than and until than, don't touch those files since they will be removed anyway?

Reply:
I've removed the new resources from RS1 as discussed in Scrum.